instrumentation/netty-reactor-0.8.0/src/main/java/reactor/netty/http/server/HttpServerMetricsHandler.java

```diff
@@ -0,0 +1,7 @@
+package reactor.netty.http.server;
+
+import com.newrelic.api.agent.weaver.SkipIfPresent;
+
+@SkipIfPresent
```

Comment:
What's going on here?

Reply:
In reactor-netty versions prior to 0.9.0, LambdaSubscriber::currentContext is implemented in the super class, which we can't weave so we needed to split the instrumentation. This is to prevent netty-reactor-0.8 from applying to 0.9+ and effectively double weaving.